### PR TITLE
build: use mach for instruments build

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

This switches the build pipeline for building instruments to use https://github.com/Synaptic-Simulations/mach/ instead of rollup and a custom plugin.

Included are also fixes for some issues that trip `esbuild` up.

* [x] Introduce new build system
* [x] Remove old build system (keep ace build scripts for now) 
* [x] Write documentation

## Testing instructions

* Make sure ALL instruments work properly
* Make sure no negative performance difference exists compared to dev

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by frankkopp** [comment]
Did a quick test - 
image
This is awesome!!

**Review by frankkopp — APPROVED** [review]
LGTM - will do another test run later

**Comment by frankkopp** [comment]
Leaving this note here for potential testers:
- flyPad still has some issues with colors and symbols due to what I think is an issue with the tailwind config
This is know and being looked at.

**Review by frankkopp — APPROVED** [review]

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #7748
Tier of Testing  : 1
Date             : 18/02/2023

Testing Process: 
Make sure ALL instruments work properly - **OK**
Make sure no negative performance difference exists compared to dev - **OK**

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

**Comment by auroraisluna** [comment]
🙏 Godsend

File: fbw-a32nx/src/systems/instruments/src/EWD/instrument.tsx [hunk 1]
```diff
@@ -117,6 +117,12 @@ class A32NX_EWD extends BaseInstrument {
         this.simVarPublisher.subscribe('slatsPositionRaw');
 
         FSComponent.render(<EwdComponent bus={this.bus} instrument={this} />, document.getElementById('EWD_CONTENT'));
+
+        // Remove "instrument didn't load" text
+        document.getElementById('EWD_CONTENT').querySelector(':scope > h1').remove();
+
+        const sub = this.bus.getSubscriber<ClockEvents>();
+        sub.on('realTime').handle((deltaTime) => this.pseudoFwc.onUpdate(deltaTime));
```
Comment (twasono) [inline fbw-a32nx/src/systems/instruments/src/EWD/instrument.tsx:125]: Just curious. Why is this line re-added, while it had been removed to fix some delta time issue in this PR https://github.com/flybywiresim/a32nx/pull/7801

Comment (tracernz) [reply fbw-a32nx/src/systems/instruments/src/EWD/instrument.tsx:125]: Indeed, that was a mistake, and even made it worse as now the FWC update method is called twice. Fixed now https://github.com/flybywiresim/a32nx/pull/7811

**Pull request merged**